packages/shared/lib/core/nfts/utils/parseNftMetadata.ts

```diff
@@ -57,7 +62,7 @@ function validate(data: IIrc27Metadata): void {
 }
 
 function validateRoyalties(royalties: unknown): void {
-    const isKeysValid = Object.keys(royalties).every((key) => !validateBech32Address(get(networkHrp), key))
+    const isKeysValid = Object.keys(royalties).every((key) => validateBech32Address(get(networkHrp), key))
     if (!isKeysValid) {
         throw `Invalid royalty address, must be a valid ${get(networkHrp)} address where royalties will be sent to.`
     }
```

Comment:
i think here we need a try catch instead of  `.every` because `validateBech32Address`  wont return any boolean anymore